### ISSUE TITLE
Angular2 project is failing tests with Karma/PhantomJS and use of `util.findChild`

### ISSUE THREAD

**Issue by donskifarrell** [OnsenUI/OnsenUI#1700]
[](Github issues should be used only for bugs and feature requests. For questions please visit https://stackoverflow.com/tags/onsen-ui or https://community.onsen.io. For bugs please use the following template.)

__Environment__ 

```
[Core]
    Onsen UI 2.0.3

[Framework]
    Angular 2.1.2

[Framework binding]
    angular2-onsenui 1.0.0-rc.1

[Additional libraries]
    None

[Platform]
    Desktop - PhantomJS 2.1.1

[Browser]
    Desktop - PhantomJS 2.1.1
```

__Encountered problem__
We are running our tests using Karma and PhantomJS, which fail with the below error:

```
PhantomJS 2.1.1 (Mac OS X 0.0.0)
TypeError: undefined is not an object (evaluating 'element.children.length')
at http://localhost:9876/base/tools/spec-bundle.js?4e73484267e018272cc1366997c1dcac91a61f11:39303
```

The bit of guilty code is found in the [Utils](https://github.com/OnsenUI/OnsenUI/blob/master/core/src/ons/util.js) file, line 50:
```

/**
 * @param {Element} element
 * @param {String/Function} query dot class name or node name or matcher function.
 * @return {HTMLElement/null}
 */
util.findChild = (element, query) => {
  const match = util.prepareQuery(query);

  for (let i = 0; i < element.children.length; i++) {
    const node = element.children[i];
    if (match(node)) {
      return node;
    }
  }
  return null;
};
```

The `element` parameter console.log is:
```
<svg class="progress-circular">&#10;    
  <circle class="progress-circular__secondary" cx="50%" cy="50%" r="40%" fill="none"
  stroke-width="10%" stroke-miterlimit="10">
  </circle>&#10; 
   
  <circle class="progress-circular__primary" cx="50%" cy="50%" r="40%" fill="none"
  stroke-width="10%" stroke-miterlimit="10">
  </circle>&#10;  
</svg>
```
The `query` parameter console.log is:
```
.progress-circular__primary
```

__How to reproduce__
[](If possible provide a working example where the issue is reproduced. For this, go to http://tutorial.onsen.io/ , select a framework and a template and modify or replace it. This will use the latest Onsen UI version released. After you finish, click 'Export to Codepen' button, save it and link it here.)

**Comment by argelius** [comment]
@donskifarrell Hi!

I think this is an issue in PhantomJS: https://github.com/ariya/phantomjs/issues/11958

It looks like it's not possible to access the `children` property for SVG elements in PhantomJS. Maybe you can try adding the shim suggested in the issue.

**Comment by donskifarrell** [comment]
Thanks for the issue link.

The last [comment](https://github.com/ariya/phantomjs/issues/11958#issuecomment-244125097) in that issue suggests using `childNodes` instead of `children`. 
Can the OnsenUI library follow that advice as well to avoid others running into this issue?

**Comment by asial-matagawa** [comment]
@donskifarrell 
We can follow the advice if it is easy to implement.
But it may require a certain amount of time due to our schedule.
Anyway, thank you for suggestion!

**Comment by asial-matagawa** [comment]
This issue was solved together with #1860.
Closing this.
